src/composables/use-pool.ts

```diff
@@ -147,6 +149,28 @@ export const usePool = (poolId: number) => {
     }
   }
 
+  const humanize_rewards = (reward: BigNumber) => {
```

Comment:
not appropriate naming case

move to helpers if can be reused somewhere

Reply:
done